src/query/service/src/sql/planner/plans/insert.rs

```diff
@@ -23,7 +24,8 @@ pub enum InsertInputSource {
     #[serde(skip)]
     SelectPlan(Box<Plan>),
     // From outside streaming source
-    StreamingWithFormat(String),
+    #[serde(skip)]
```

Comment:
Can we remove `serde::Serialize, serde::Deserialize` for `InsertInputSource`? Do we really need this?

Reply:
no idea， maybe for distributed execute？ cc @zhang2014 @sundy-li

Reply:
still ok here because StreamingWithFormat can only run in one node.

Reply:
```
#[derive(serde::Serialize, serde::Deserialize, Clone)]
pub struct Insert {
   ...
    pub source: InsertInputSource,
}

#[derive(Clone, Debug)]
pub enum Plan {
  ... 
  Insert(Box<Insert>),
}
```
but enum Plan  not impl Serialize